test/unit/printer/sourcemaps.spec.ts

```diff
@@ -86,9 +86,8 @@ test.each([
         assertPatterns: [
             { luaPattern: "Bar = __TS__Class()", typeScriptPattern: "class Bar" },
             { luaPattern: "Bar.name =", typeScriptPattern: "class Bar" },
-            { luaPattern: "Bar.____super = Foo", typeScriptPattern: "Foo {" },
-            { luaPattern: "setmetatable(Bar,", typeScriptPattern: "Foo {" },
-            { luaPattern: "setmetatable(Bar.prototype,", typeScriptPattern: "Foo {" },
+            { luaPattern: "__TS__ClassExtends(", typeScriptPattern: "extends" },
+            { luaPattern: "Foo)", typeScriptPattern: "Foo {" },
```

Comment:
This last pattern seems a bit strange to test. Could we instead test some other lua pattern we actually care about, otherwise just delete this. I doubt lua will ever throw an error on `Foo)`'s location anyway.

Reply:
I've changed it to a function call, which should be a more important mapping